packages/cli/web/index.js

```diff
@@ -8,31 +8,52 @@ const mustacheExpress = require("mustache-express")
 const credsPath = path.join(os.homedir(), ".rb")
 
 /**
- * Routing handler helper
- * @param {boolean} ciServer indicates if the connection is throw a ci server
- * @param {object} req - request obj
- * @param {object} res - request res
- * @param {string} state - state number for validation propose
+ * Check if creds are existed , if not write them again
```

Comment:
If you use the normal LoginCommand - you should be able to remove any code that you added to the server. 

Or at least most of it.